docs/getting-started-apm-server.asciidoc

```diff
@@ -0,0 +1,296 @@
+If you are running Windows XP, you may need to download and install PowerShell.
+
+. From the PowerShell prompt, run the following commands to install APM Server as a Windows service:
++
+[source,shell]
+----------------------------------------------------------------------
+PS > cd 'C:\Program Files\APM-Server'
+PS C:\Program Files\APM-Server> .\install-service-apm-server.ps1
+----------------------------------------------------------------------
+
+NOTE: If script execution is disabled on your system,
+you need to set the execution policy for the current session to allow the script to run.
+For example: `PowerShell.exe -ExecutionPolicy UnRestricted -File .\install-service-apm-server.ps1`.
+
+endif::[]
+
+[[docker]]
+*Docker:*
+
+See <<running-on-docker, Running on Docker>> for deploying Docker containers.
+
+// *******************************************************
+// STEP 2
+// *******************************************************
+
+[[apm-server-configuration]]
+=== Step 2: Configure APM Server
+
+If you've installed APM Server yourself,
+you can edit the `apm-server.yml` configuration file to customize APM Server to your needs.
+The location of this file varies by platform. See <<directory-layout>> for help with locating it.
+
+All available configuration options are outlined in
+{apm-server-ref-v}/configuring-howto-apm-server.html[configuring APM Server].
+
+[float]
+[[index-template-config]]
+==== Index template configuration
+
+IMPORTANT: The {kibana-ref}/xpack-apm.html[Kibana APM UI] relies on specific mappings.
+Changing the default mapping might break the APM UI.
+
+In Elasticsearch, {ref}/indices-templates.html[index templates]
+are used to define field settings and mappings.
+
+The recommended index template file for {beatname_uc} is installed by the {beatname_uc} packages.
+If you accept the default configuration in the +{beatname_lc}.yml+ configuration file,
+{beatname_uc} will load the template automatically after successfully connecting to Elasticsearch.
+If a template already exists, it will not be overwritten unless you configure {beatname_uc} to do so.
+Users can run `apm-server -e setup template` to force a template overwrite.
+
+To see the template bundled with {beatname_uc}, run `apm-server export template`.
+You can then compare the response with the actual template in your elasticsearch instance.
+
+NOTE: Index templates cannot be applied retroactively.
+They are only applied at index creation time.
+Changing a template will have no impact on existing indices.
+This means you need to ensure the correct index template is loaded from day one.
+
+More information is available in <<load-template-auto, configure template loading>>.
+
+// *******************************************************
+// STEP 3
+// *******************************************************
+
+[[apm-server-starting]]
+=== Step 3: Start APM Server
```

Comment:
Sorry for the late response here, but I do actually think that describing to run `apm-server setup` before starting the APM Server is a good idea. One can apply config values the same way as when starting the server, but might see config issues or failing requirements early on. It can be helpful to ensure everything is setup properly before starting data ingestion.

Reply:
In my first attempt at this, I've decided to turn step 2 into "Setup and Configure". I've then used a tagged region to pull in the `setup` command from the command reference page. You can see the changes in 860696c or here: http://apm-server_2170.docs-preview.app.elstc.co/guide/en/apm/server/master/apm-server-configuration.html

Reply:
LGTM